src/main/resources/templates/pages/featureOverview.vm

```diff
@@ -1,6 +1,9 @@
 <!DOCTYPE html>
 <html xmlns="http://www.w3.org/1999/xhtml">
 <head>
+#if($fromJenkins)
+	<base href="${jenkins_base}job/$build_project/$build_number/cucumber-html-reports/">
```

Comment:
Both are possible of course. But the report in the mail is much more appealing to the user/team that is responsible for the feature(s). Our experience is that the link isn't always clicked on and with the report in the mail there is no way around it.